boards/pico/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#![no_std]
+
+extern crate rp2040_hal as hal;
+
+#[cfg(feature = "rt")]
+extern crate cortex_m_rt;
+#[cfg(feature = "rt")]
+pub use cortex_m_rt::entry;
+
+pub use hal::pac;
+
+hal::bsp_pins!(
```

Comment:
Are these definitions correct? The `feather_rp2040` BSP defines all pins, not just the ones with dedicated functions.
I'm not sure if this is necessary, though.

Reply:
Ah you might be right!